test/include/seqan3/test/pretty_printing.hpp

```diff
@@ -12,7 +12,9 @@
 
 #pragma once
 
-#include<seqan3/core/debug_stream.hpp>
+#include <ios>
+
+#include <seqan3/core/debug_stream.hpp>
```

Comment:
hmm... so `#include <seqan3/core/debug_stream.hpp>` still pulls in the "whole library" right? Then pretty printing does, which we use throughout the tests, so we did not gain the desired independence of unit tests do we? :see_no_evil:

Reply:
Yes, but this PR already improves it quite a lot. I would like to make a new PR to fix it for pretty_printing.